src/preact/base-element.js

```diff
@@ -892,7 +916,9 @@ function shouldMutationBeRerendered(Ctor, m) {
       const def = /** @type {!AmpElementPropDef} */ (props[name]);
       if (
         m.attributeName == def.attr ||
-        (def.attrs && def.attrs.includes(devAssert(m.attributeName)))
+        (def.attrs && def.attrs.includes(devAssert(m.attributeName))) ||
+        (m.attributeName.startsWith(def.attrPrefix) &&
```

Comment:
We should probably check `def.attrPrefix` exists before trying to match prefix. Obviously super unlikely that someone would have an attribute "undefined-x", but still better to keep it tight.